Runtime/AvatarCreator/UI/SelfieElement.cs

```diff
@@ -0,0 +1,55 @@
+using System;
```

Comment:
Should this element be named 'SelfieToAvatarElement?', since it's so deeply tied to creating an avatar?

Reply:
Yep ➕ to this, seems more descriptive.